packages/hdwallet-ledger/src/utils.ts

```diff
@@ -151,7 +156,12 @@ export const networksUtil = {
       bech32: "bc",
       bip32: {
         private: 70615956,
-        public: 70617039
+        public: {
+          p2pkh: 76067358,
```

Comment:
need to make all the coins in this table have the same format.

Reply:
:+1: